.github/workflows/daily-e2e-tests-ibmcloud.yaml

```diff
@@ -23,6 +23,7 @@ jobs:
           - type: amd64
           - type: s390x-non-secure-execution
           - type: s390x-secure-execution
+          - type: amd64-csi-wrapper
```

Comment:
would you like use a different keyword amd64 here, it will make the pipeline use a wrong log file
maybe use:
```
          - type: csi-wrapper-x86_64
```